LICENSE.md

```diff
@@ -0,0 +1,8 @@
+Copyright 2024 TRENZ GmbH
```

Comment:
do we really need _both_ a `LICENSE` and a `LICENSE.md`?

Reply:
Oh I didn't see the other file since it didn't appear in the solution items...